npm-scripts.js

```diff
@@ -1,177 +1,209 @@
-/* eslint-disable no-console */
-
 const process = require('process');
 const os = require('os');
 const fs = require('fs');
 const { execSync, spawnSync } = require('child_process');
-const { version } = require('./package.json');
+const fetch = require('node-fetch');
+const tar = require('tar');
+
+const PKG = JSON.parse(fs.readFileSync('./package.json').toString());
+const IS_FREEBSD = os.platform() === 'freebsd';
+const IS_WINDOWS = os.platform() === 'win32';
+const MAYOR_VERSION = PKG.version.split('.')[0];
+const MAKE = process.env.MAKE || (IS_FREEBSD ? 'gmake' : 'make');
+const WORKER_RELEASE_DIR = 'worker/out/Release';
+const WORKER_RELEASE_BIN = 'mediasoup-worker';
```

Comment:
I've had challenges with this on windows, the bin is `mediasoup-worker.exe`. 

In my pr, [I had done some searching](https://github.com/versatica/mediasoup/pull/1083/files#diff-0e4b0cf2dfea5eb3fd7fcefa330bbcbe8ec35f53426f00d14f34a94281ed7a2aR75-R83) in the package state to find all of the files at the top level, which I'm not sure is right either.

```suggestion
const WORKER_RELEASE_BIN = `mediasoup-worker${IS_WINDOWS ? '.exe' : ''}`;
```

I think this will fix that problem in a more obvious way.

Reply:
Thanks.